modules/virt-about-ssp-operator.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// * virt/virt-architecture.adoc
+// * virt/about_virt/virt-architecture.adoc
 
 :_content-type: CONCEPT
 [id="virt-about-ssp-operator_{context}"]
 = About the Scheduling, Scale, and Performance (SSP) Operator
 
-The SSP Operator, `ssp-operator`, deploys the common templates, the related default boot sources, and the template validator.
+The SSP Operator, `ssp-operator`, deploys the common templates, the related default boot sources, the tekton tasks, and the template validator.
```

Comment:
I am fairly certain that tekton is (a) capitalized and (b) an upstream term. It was Tekton pipelines before it became OpenShift pipelines. So "tekton tasks" should probably be "pipeline tasks". You can ask the SME.

Reply:
confirmed with me that it should be "pipeline tasks", but they also mentioned that "[...]but we deploy both Tekton Tasks and Tekton Pipelines". for now, changing tekton tasks to pipeline tasks